intuita-webview/src/shared/Snippet/DiffEditor.tsx

```diff
@@ -1,28 +1,25 @@
 import { forwardRef } from 'react';
 import { DiffEditorProps, DiffEditor } from '@monaco-editor/react';
-import * as monaco from 'monaco-editor';
-
-export { monaco };
+import type { editor } from 'monaco-editor';
 
 type DiffViewerProps = DiffEditorProps & { onRefSet: () => void };
-const DiffViewer = forwardRef<
-	monaco.editor.IStandaloneDiffEditor,
-	DiffViewerProps
->(({ onRefSet, ...props }, ref) => {
-	return (
-		<div className="relative w-full h-full">
-			<DiffEditor
-				onMount={(editor) => {
-					typeof ref === 'function' && ref(editor);
-					if (!(typeof ref === 'function') && ref) {
-						ref.current = editor;
-					}
-					onRefSet();
-				}}
-				{...props}
-			/>
-		</div>
-	);
-});
+const DiffViewer = forwardRef<editor.IStandaloneDiffEditor, DiffViewerProps>(
```

Comment:
did only formatting change here?

Reply:
changed from `monaco.editor.IStandaloneDiffEditor` to `editor.IStandaloneDiffEditor`, i think when we import with `"*" as monaco`, it bundles the whole module.